app/frontend/src/aquifers/legend.js

```diff
@@ -0,0 +1,24 @@
+export function buildLegendHTML (items) {
```

Comment:
What's the reason behind doing it this way vs creating a vue template?

Reply:
Yes. This pattern follows what was done before (and it was duplicated between AquiferMap.vue and SingleAquiferMap.vue). I attempted to use a Vue component for the legend but after looking for examples online and using the leaflet2-vue project as a guide there was no "good" way of doing it. It doesn't seem to me that Vue is supposed to be integrated into non-Vue DOM - at least not officially. This function is at least an improvement on what was there as it removes duplication and produces cleaner HTML.